packages/contracts-bedrock/test/safe/LivenessModule2.t.sol

```diff
@@ -8,9 +8,71 @@ import "test/safe-tools/SafeTestTools.sol";
 import { LivenessModule2 } from "src/safe/LivenessModule2.sol";
 import { SaferSafes } from "src/safe/SaferSafes.sol";
 
+import { ModuleManager } from "safe-contracts/base/ModuleManager.sol";
+
+/// @title LivenessModule2_TestUtils
+/// @notice Reusable helper methods for LivenessModule2 tests.
+abstract contract LivenessModule2_TestUtils is Test, SafeTestTools {
```

Comment:
Created this so that it can be used both here and in SaferSafes.t.sol. Not sure, but making it abstract hopefully reduces compile times.